client/signup/steps/site-or-domain/index.jsx

```diff
@@ -201,9 +201,8 @@ class SiteOrDomain extends Component {
 
 		// we can skip the next two steps in the `domain-first` flow if the
 		// user is only purchasing a domain
-		this.props.submitSignupStep( { stepName: 'site-picker', wasSkipped: true } );
 		this.props.submitSignupStep(
-			{ stepName: 'themes', wasSkipped: true },
+			{ stepName: 'site-picker', wasSkipped: true },
 			{ themeSlugWithRepo: 'pub/twentysixteen' }
```

Comment:
Might be wrong here but wondering why we need to keep the themeSlugWithRepo thing, should it not be something like this:

```suggestion
			{ stepName: 'site-picker', wasSkipped: true }
```

Reply:
Or is it because the next step needs that dependency regardless of where it came from?

Reply:
Yes without the dependency this flow breaks :)